next
thenReturn
getInternalServerErrorResult
getFilterChain
testOnRouteRequestWhenInternalServerErrorException
filterChain
when
ninjaDefault
route
Mockito
contextImpl
internalServerErrorException
mock
verify
That's an InternalServerErrorException that should be handled by onError!
onRouteRequest
thenThrow
